packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -270,10 +306,31 @@ abstract class ImageProvider<T> {
   /// This is the public entry-point of the [ImageProvider] class hierarchy.
   ///
   /// Subclasses should implement [obtainKey] and [load], which are used by this
-  /// method.
+  /// method. If they need to change the implementation of [ImageStream] used,
+  /// they should override [createStream]. If they need to manage the actual
+  /// resolution of the image, they should override [resolveStreamForKey].
```

Comment:
Maybe add a link to the lifecycle explanation of this method in the class comment?

Reply:
Done - I think.